processing/data_collection/gazette/spiders/pr_sao_jose_pinhais.py

```diff
@@ -0,0 +1,45 @@
+import re
+import dateparser
+import w3lib.url
+
+from datetime import datetime
+from scrapy import Request, Spider
+from gazette.items import Gazette
+
+
+class PrSaoJosePinhaisSpider(Spider):
+    GAZETTE_ELEMENT_CSS = '.container-publicacao .item-publicacao'
+    DATE_CSS = '.item-info::text'
+    NEXT_PAGE_CSS = '.item-paginacao a:last-child::attr(href)'
+
+    MUNICIPALITY_ID = '4125506'
+
+    allowed_domains = ['diariooficial.sjp.pr.gov.br']
+    name = 'pr_sao_jose_pinhais'
+    start_urls = ['http://diariooficial.sjp.pr.gov.br/?ano_edicao=&entidade=12526&pg=1']
+
+    def parse(self, response):
+        """
+        @url http://diariooficial.sjp.pr.gov.br/?ano_edicao=&entidade=12526&pg=1
+        @returns requests 1
+        @scrapes date file_urls is_extra_edition municipality_id power scraped_at
+        """
+
+        for element in response.css(self.GAZETTE_ELEMENT_CSS):
+            url = element.css('a::attr(href)').extract_first()
+            date = dateparser.parse(element.css(self.DATE_CSS).extract()[2], languages=['pt']).date()
+
+            yield Gazette(
+                date=date,
+                file_urls=[url],
+                is_extra_edition=False,
```

Comment:
We don't have a extra edition of this city yet, right?

Reply:
Opened several editions here and didn't find any mention about extra editions...